unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/ChatWidgetHUD/ChatUtils.cs

```diff
@@ -0,0 +1,10 @@
+public static class ChatUtils
+{
+    public static string AddNoParse(string message)
+    {
+        var filteredMessage = message.Replace("noparse", "");
```

Comment:
this won't replace the text if the user is saying "noparse" in the chat?

Reply:
@AjimenezDCL should we replace "<noparse>" and "</noparse>" instead?

Reply:
sure, the chances of somebody using "noparse" in a real sentences are quite slim, but I will change it anyway

Reply:
I'm going to replace:

`<noparse`  and `</noparse`.

Because othrwise we should replade: `<noparse>`, `<noparse >`, `</noparse>` and `</noparse >`